guides/common/modules/con_updating-server-and-smart-proxy.adoc

```diff
@@ -0,0 +1,9 @@
+[id="updating-server-and-smart-proxy_{context}"]
+= Updating {ProjectServer} and {SmartProxyServerTitle}
```

Comment:
```suggestion
= Updating {Project} to the Next Minor Version
```
Also rename the file and ID accordingly.

Reply:
Addressed :+1: